src/core/viewports/web-mercator-viewport.js

```diff
@@ -267,14 +267,16 @@ export default class WebMercatorViewport extends Viewport {
 
   // TODO - should support user supplied constraints
   isMapSynched() {
+    const EPSILON = 0.000001;
     const MAPBOX_LIMITS = {
       pitch: 60,
       zoom: 40
     };
 
     const {pitch, zoom} = this;
 
-    return pitch <= MAPBOX_LIMITS.pitch && zoom <= MAPBOX_LIMITS.zoom;
+    return pitch <= (MAPBOX_LIMITS.pitch + EPSILON) &&
```

Comment:
Normally one does something like `Math.abs(a - b) < EPSILON` to make sure signs etc don't affect the results. As one option. math.gl exports an `equals` function that does this.

Reply:
We probably should add lessOrEqual, greatOrEqual version to math as well.